src/profile/components/editProfileForm/EditProfileForm.tsx

```diff
@@ -479,13 +497,24 @@ function EditProfileForm(props: Props) {
                             )}
                           />
 
-                          <Field
+                          <FormikDropdown
                             className={styles.formField}
-                            as={Select}
                             name={`addresses.${index}.countryCode`}
                             id={`addresses.${index}.countryCode`}
                             options={countryOptions}
-                            labelText={t('profileForm.country')}
+                            label={t('profileForm.country')}
+                            default={
+                              formikProps.values.addresses[index].countryCode
+                            }
+                            // onChange option type needs to match HDS that's why its set to OptionType | OptionType.
```

Comment:
I guess you mean `OptionType | OptionType[]` 

Suggested even shorter comment: 

// onChange is set to OptionType | OptionType[] in order to match hds typing

Reply:
Changed comments to what you suggested. But yeah, the `onChange` is kinda tricky, because of the typing it's hard to make it more generic than this.

Reply:
You could also consider if you wanted to force the typing. The reason for why the types are like so is that the HDS dropdown supports multiselect. Your implementation does not. You could hard code your implementation to be "single select" by not allowing the user to define the flag that turns the control into a multiselect. After that you could only have `OptionType` safely. You could likely enforce that type in the in the formik binding component.

You could then, when you need it, define a multi select binding for formik later on.

Reply:
Setting `multiselect` to false actually did the trick. Thanks!